modules/cosco/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/CoSCO",
+	"url": "https://www.dndbeyond.com/sources/dnd/cosco",
```

Comment:
This URL 404's as well, but I'm not sure what the correct one is. All I can find by searching the DDB site is this, which doesn't seem right: `https://www.dndbeyond.com/sources/dnd/cos/appendix-a-character-options#AppendixACharacterOptions`.